jax/experimental/global_device_array.py

```diff
@@ -189,86 +188,85 @@ class GlobalDeviceArray:
   devices, and the data is visible to the current process. Global shards are
   those across all devices (including local devices), and the data isn’t visible
   if the shard is on a non-local device with respect to the current process.
-  Please see the `Shard` class to see what information is stored inside that
+  Please see the ``Shard`` class to see what information is stored inside that
   data structure.
 
   Note: to make pjit output GlobalDeviceArrays, set the environment variable
-  `JAX_PARALLEL_FUNCTIONS_OUTPUT_GDA=true` or add the following to your code:
-  `jax.config.update('jax_parallel_functions_output_gda', True)`
+  ``JAX_PARALLEL_FUNCTIONS_OUTPUT_GDA=true`` or add the following to your code:
+  ``jax.config.update('jax_parallel_functions_output_gda', True)``
+
+  Args:
+    global_shape: The global shape of the array.
+    global_mesh: The global mesh representing devices across multiple
+      processes.
+    mesh_axes: A sequence with length less than or equal to the rank of the
+      global array (i.e. the length of the global shape). Each element can be:
+
+      * An axis name of ``global_mesh``, indicating that the corresponding
+        global array axis is partitioned across the given device axis of
+        ``global_mesh``.
+      * A tuple of axis names of ``global_mesh``. This is like the above option
+        except the global array axis is partitioned across the product of axes
+        named in the tuple.
+      * None indicating that the corresponding global array axis is not
+        partitioned.
+
+      For more information, please see:
+      https://jax.readthedocs.io/en/latest/jax-101/08-pjit.html#more-information-on-partitionspec
+    device_buffers: DeviceArrays that are on the local devices of ``global_mesh``.
 
   Attributes:
-    shape: The global shape of the array.
-    dtype: dtype of the global array.
-    local_shards: List of `Shard`s on the local devices of the current process.
-      Data is available for all local shards.
-    global_shards: List of all `Shard`s of the global array. Data isn’t
+    shape: Global shape of the array.
+    dtype: Dtype of the global array.
+    local_shards: List of ``Shard`` on the local devices of the current process.
```

Comment:
You can make `Shard` a link to the Shard definition. I think the following might work:
```
:class:`Shard`
```

https://stackoverflow.com/questions/7753805/sphinx-automodule-how-to-reference-classes-in-same-module might also be useful for figuring this out.

Reply:
Done